service/src/main/java/bio/terra/externalcreds/services/ProviderService.java

```diff
@@ -184,12 +184,12 @@ public void deleteLink(String userId, String providerId) {
     linkedAccountService.deleteLinkedAccount(userId, providerId);
   }
 
-  public int validatePassportsWithAccessTokenVisas() {
-    var passportDetailsList = passportService.getPassportsWithUnvalidatedAccessTokenVisas();
+  public int validateAccessTokenVisas() {
+    var visaDetailsList = passportService.getUnvalidatedAccessTokenVisaDetails();
 
-    passportDetailsList.forEach(
+    visaDetailsList.forEach(
```

Comment:
I would actually change this implementation a bit. The problem is that a passport may have multiple visas and if more than one is invalid we refresh the entire passport more than once. So I would change this to 2 steps. First find all the linked accounts that need refreshing:
```
    var linkedAccountIdsToRefresh = visaDetailsList.stream().flatMap(visaDetails -> {
      var responseBody = validateVisaWithProvider(visaDetails);
      return responseBody.equalsIgnoreCase("valid") ? Stream.empty() : Stream.of(visaDetails.getLinkedAccountId());
    }).distinct();
```
Then for each of those do the block of code starting at line 196 and log each linked account id being refreshed